package.json

```diff
@@ -48,7 +48,7 @@
     "vuexfire": "^3.2.4"
   },
   "devDependencies": {
-    "@firebase/testing": "^0.20.11",
+    "@firebase/rules-unit-testing": "^1.0.7",
```

Comment:
`@firebase/testing` is deprecated, so I changed it to the new dependency `@firebase/rules-unit-testing`